overlays/fixed-2clusters/kustomization.yaml

```diff
@@ -0,0 +1,13 @@
+bases:
+  - ../../release
+patches:
+  - adservice.yaml
```

Comment:
I should be possible to define only the two generic patches, and then select which resources they are applied to. An example is [here](https://github.com/kubernetes-sigs/kustomize/blob/master/examples/patchMultipleObjects.md), even though you may need to add a different entry for each resource (since there is no way to select all of them). Still, this makes it easier to understand to which service you apply what, and reduces the total amount of manifests to maintain.